src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/domene/VilkårResultat.kt

```diff
@@ -95,6 +95,9 @@ class VilkårResultat(
 
     fun erIkkeAktuelt() = this.resultat == Resultat.IKKE_AKTUELT
 
+    fun harFullBarnehageplass() =
+        this.søkerHarMeldtFraOmBarnehageplass == true && (this.antallTimer == null || this.antallTimer >= BigDecimal(33))
```

Comment:
```suggestion
        this.søkerHarMeldtFraOmBarnehageplass == true && this.antallTimer != null && this.antallTimer >= BigDecimal(33)
```

Reply:
`this.antallTimer == null` betyr at de ikke har barnehageplass og skal få 100% av ytelsen

Reply:
Da blir logikken feil? 
Hvis antallTimer == null og søkarHarMeldtFraOmBarnehageplass == true så vil vi returnere true?

Reply:
Per nå heter funksjonen harFullBarnehageplass.
Men man kan vel ha fullBarnehageplass selvom søker ikke har meldt ifra om barnehageplass (altså krysset av for den checkboxen i frontend).